### ISSUE TITLE
incompatibilities with an up-to-date project

### ISSUE THREAD

**Issue by travi** [mthuret/storybook-addon-specifications#95]
## React

When using this addon in a plugin that has been upgraded to react v16, we get the following peer dependency warnings:

```
npm ERR! peer dep missing: react@^0.14.7 || ^15.0.0, required by storybook-addon-specifications@2.1.1
npm ERR! peer dep missing: react-dom@^0.14.7 || ^15.0.0, required by storybook-addon-specifications@2.1.1
```

## Addons Channel

Worse though, if using the latest version of `@storybook/react`, all tests run through this addon fail with the following error:

```
     Error: Accessing nonexistent addons channel, see https://storybook.js.org/basics/faq/#why-is-there-no-addons-channel
```

based on [this issue](https://github.com/storybooks/storybook/issues/1981) is because all addons need to point to the same version of `@storybook/addons`. i agree that [something needs to change](https://github.com/storybooks/storybook/issues/1981#issuecomment-345193670) so that addon versions aren't so coupled to each other through the `@storybook/addons` package, but it looks like that is the case for now. i did confirm that this addon is the only one in my project that is not up to date with the `@storybook/addons` package

<details>
  <summary>Full Stack Trace:</summary>
```
     Error: Accessing nonexistent addons channel, see https://storybook.js.org/basics/faq/#why-is-there-no-addons-channel
      at AddonStore.getChannel (node_modules/@storybook/addons/dist/index.js:39:15)
      at nesting depth (node_modules/@storybook/addon-actions/dist/preview.js:38:36)
      at PdfTable (src/table/atoms/pdf.js:8:3)
      at mountIndeterminateComponent (node_modules/react-dom/cjs/react-dom.development.js:8032:15)
      at beginWork (node_modules/react-dom/cjs/react-dom.development.js:8221:16)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:10224:16)
      at workLoop (node_modules/react-dom/cjs/react-dom.development.js:10288:26)
      at Object.invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:491:10)
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:438:27)
      at renderRoot (node_modules/react-dom/cjs/react-dom.development.js:10366:7)
      at performWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:11014:24)
      at performWork (node_modules/react-dom/cjs/react-dom.development.js:10967:7)
      at requestWork (node_modules/react-dom/cjs/react-dom.development.js:10878:7)
      at scheduleWorkImpl (node_modules/react-dom/cjs/react-dom.development.js:10732:11)
      at scheduleWork (node_modules/react-dom/cjs/react-dom.development.js:10689:12)
      at scheduleTopLevelUpdate (node_modules/react-dom/cjs/react-dom.development.js:11193:5)
      at Object.updateContainer (node_modules/react-dom/cjs/react-dom.development.js:11231:7)
      at node_modules/react-dom/cjs/react-dom.development.js:15226:19
      at Object.unbatchedUpdates (node_modules/react-dom/cjs/react-dom.development.js:11102:12)
      at renderSubtreeIntoContainer (node_modules/react-dom/cjs/react-dom.development.js:15225:17)
      at Object.render (node_modules/react-dom/cjs/react-dom.development.js:15290:12)
      at Object.render (node_modules/enzyme-adapter-react-16/build/ReactSixteenAdapter.js:218:50)
      at new ReactWrapper (node_modules/enzyme/build/ReactWrapper.js:98:16)
      at mount (node_modules/enzyme/build/mount.js:19:10)
      at Context.<anonymous> (src/table/organisms/stories.js:286:25)
```
</details>

## New Version?

it looks like [this commit](https://github.com/mthuret/storybook-addon-specifications/commit/ec4877f4a968290156a972fe632c881e0596af6b) solves both issues, but isn't published. is there anything holding back the release of a new version?

**Comment by peet86** [comment]
@mthuret Do you have any plans concerning the PRs and releasing a new version? The initial idea was a really good one.. we should keep this plugin alive. Let me know if I can help.

**Comment by mthuret** [comment]
I just published a new version (2.1.2), is it better? I lost a bit track of where we were. The 2.1.1 was not working? 

@peet86 if you want, I can add you to the team of this project. It's true that lately I don't have much time to continue monitor it. Let me know :)

**Comment by jonwilliams-bluescape** [comment]
Should be working now
